### ISSUE TITLE
Game Grid View doesn't have keyboard focus when Playnite starts-up

### ISSUE THREAD

**Issue by SAH7777** [JosefNemec/Playnite#2792]
### Bug Description

I just started using Playnite a few days ago and am so far liking it, thanks! But I'm having a problem where when Playnite starts-up (Desktop mode), the Grid View area (the area that has all my game 4:3 cover images/box art) doesn't have keyboard focus, and so the arrow keys on my keyboard don't move the game selection box around. 

I'm not sure if this is technically a bug or not, but I find that I have to mouse-click somewhere in the Grid View to give it keyboard focus and for my arrow keyboard keys to start working. This is slightly annoying because I started-up Playnite with my Xbox controller (using Controller Companion) and so am holding my Xbox controller in my hands.

Thank you.

PS: On Playnite help forum, user Crow said/replied "Desktop mode is not designed to be used with a controller or keyboard exclusively so these issues are to be expected. You can open issue on GitHub for it and I'll look into it later. I don't know what element gets default focus on startup, but you should be able to use Tab key to move to the game list."

### To Reproduce

Open Playnite (Desktop Mode and Grid View) and then immediately try using keyboard arrow keys to move the game selection box around (it fails to move for me)

### Diagnostics ID

200b15f6-c9f3-42e2-9532-5e32ab6cd6a5

**Comment by heinkhant4ung** [comment]
Please fix this bug
